cypress/components/action-popover/action-popover.cy.tsx

```diff
@@ -1,1197 +0,0 @@
-    });
-
-    it.each([
-      [true, "right"],
-      [false, "left"],
-    ])(
-      "should render ActionPopover with rightAlignMenu set to %s",
-      (rightAlignMenu, css) => {
-        CypressMountWithProviders(
-          <ActionPopoverWithProps rightAlignMenu={rightAlignMenu} />
-        );
-
-        actionPopoverButton().eq(0).click();
-        actionPopover().should("have.css", css);
-      }
-    );
-
-    it("should render ActionPopover with custom button", () => {
-      CypressMountWithProviders(
-        <ActionPopoverWithProps
-          renderButton={({
-            tabIndex,
-            "data-element": dataElement,
-            ariaAttributes,
-          }: RenderButtonProps) => (
-            <ActionPopoverMenuButton
-              buttonType="tertiary"
-              iconType="dropdown"
-              iconPosition="after"
-              size="small"
-              tabIndex={tabIndex}
-              data-element={dataElement}
-              ariaAttributes={ariaAttributes}
-            >
-              More
-            </ActionPopoverMenuButton>
-          )}
-        />
-      );
-
-      buttonDataComponent().click();
-      actionPopoverWrapper()
-        .find('[data-component="button"]')
-        .should("be.visible");
-    });
-
-    it("should render ActionPopover with icons within a submenu", () => {
-      CypressMountWithProviders(<ActionPopoverWithSubmenusAndNoIcons />);
-
-      actionPopoverButton().eq(0).click();
-      actionPopoverMenuItemIcon().eq(0).should("exist");
-    });
-
-    it.each([
-      ["left", "start"],
-      ["right", "end"],
-    ])(
-      "should render ActionPopover with horizontalAlignment prop set to %s",
-      (position, attrValue) => {
-        CypressMountWithProviders(
-          <ActionPopoverWithProps horizontalAlignment={position} />
-        );
-
-        actionPopoverButton().eq(0).click();
-        actionPopover()
-          .get("button")
-          .should("have.css", "justify-content", `flex-${attrValue}`);
-      }
-    );
-
-    it.each([
-      ["left", "chevron_left_thick"],
-      ["right", "chevron_right_thick"],
-    ])(
-      "should render ActionPopover with submenuPosition prop set to %s",
-      (position, chevronType) => {
-        CypressMountWithProviders(
-          <ActionPopoverWithSubmenusAndIcons submenuPosition={position} />
-        );
-
-        actionPopoverButton().eq(0).click();
-        actionPopoverMenuItemChevron().should("have.attr", "type", chevronType);
-      }
-    );
-
-    it("should render ActionPopoverMenu with menuID", () => {
-      CypressMountWithProviders(
-        <ActionPopoverMenuWithProps menuID="cypress" />
-      );
-
-      actionPopoverButton().eq(0).click();
-      actionPopover()
-        .eq(1)
-        .should("have.attr", "id")
-        .and("contain", "ActionPopoverMenu_");
-    });
-
-    it("should render ActionPopoverMenu with focusIndex set to null", () => {
```

Comment:
It doesn't need to be covered because it's testing a prop that does nothing (because we override it internally) and that "should" be private - but that for some reason we do document in the public interface. There was some discussion on Slack of this when I found it out while doing this refactor - Ed agreed there was no reason to keep the test in. FE-6211 was raised to investigate and see if we are safe to remove the prop (I think we almost certainly are).